.circleci/config.yml

```diff
@@ -58,9 +58,9 @@ jobs:
           name: Authenticate with github
           command: |
             echo $GH_CR_TOKEN | docker login ghcr.io -u USERNAME --password-stdin
-      ## This could be its own step only for tagged builds
+        # NOTE: eventually we might only want to publish an image to GHCR for tags or certain branches
       - run:
-          name: Push to ECR
+          name: Push to GHCR
           command: |
             TAG="${CIRCLE_TAG:-${CIRCLE_SHA1}}"
             docker tag duality:local ghcr.io/duality-labs/duality:${TAG}
```

Comment:
> * is the docker-compose section necessary? from what I think I understand, the docker-compose and override allows us to generate a cloudformation template with a surprisingly small amount of docker-compose settings and a lot of magic. But we do this from a dev's machine and not from CI. So a dev can stand up a group of resources and the CI then updates them. 

This is what I was using to standup the devnet using `docker-compse up` (with ecs context)  since the CI process is not able to do this by itself. Technically it could, but simply updating the image tag on a running devnet is a much smaller task than the full cloud formation setup. Arguably this doesn't have to exist in the codebase, but it seems like a convenient place to keep it. It also makes it very easy to spin up a local devnet.